packages/cli/src/commands/setup/tailwind/tailwind.js

```diff
@@ -74,7 +74,7 @@ export const handler = async ({ force }) => {
                 'add',
                 '-D',
                 'postcss-loader@4.0.2',
-                'tailwindcss',
+                'tailwindcss@npm:@tailwindcss/postcss7-compat',
```

Comment:
I see that they use this syntax at https://tailwindcss.com/docs/installation#post-css-7-compatibility-build but it still looks weird. Why not just `@tailwindcss/postcss7-compat`? It's what npm says to use https://www.npmjs.com/package/@tailwindcss/postcss7-compat

Reply:
The fine people over in the tailwind discord helped me figure it out. It's a way to alias the package, so we can still use `import foobar from tailwindcss` even though the package name is actually something else
https://classic.yarnpkg.com/en/docs/cli/add/#toc-yarn-add-alias